services/spar/test-integration/Test/LoggingSpec.hs

```diff
@@ -2,13 +2,21 @@ module Test.LoggingSpec (spec) where
 
 import Imports
 import Control.Lens
+import Data.String.Conversions (cs)
+import Network.HTTP.Types.Status (statusCode)
+import Network.Wai (Application)
 import Spar.App
-import System.Logger as Log
+import Spar.Run (mkApp)
 import System.IO.Silently (capture)
+import System.Logger as Log
 import Util
 
+import qualified Network.Wai.Test as HW
+import qualified Test.Hspec.Wai as HW
+import qualified Test.Hspec.Wai.Internal as HW
 
-spec :: HasCallStack => SpecWith TestEnv
+
+spec :: SpecWith TestEnv
```

Comment:
Can I ask what the reasoning for this change was?

Reply:
I was hoping to get better hspec errors, but they are the same now: error happens in Spec.hs in the line where this is called.  I would like to investigate this some other time.